Update the doc of DPG guide on  'How to generate client libraries'. (#1438)
Azure/typespec-azure: docs/howtos/Client Generation/00howtogen.mdx

# How to generate client libraries

> This page outlines the steps to build a client library based on your TypeSpec specification. It provides the guide of the whole process from initial API design to the release of SDKs. please visit https://aka.ms/azsdk/dpcodegen.

Reviewer: ```suggestion
This page outlines the steps to build a client library based on your TypeSpec specification. It provides a guide for the whole process from initial API design to the release of SDKs. please visit https://aka.ms/azsdk/dpcodegen.
```

At a glance, add your emitter name to your `package.json`, along with the right options in the `tspconfig.yaml`.

> > **NOTE:** Generating all languages from the same folder may be complicated due to the preview status of the emitters and TypeSpec libraries, such as `typespec-azure-core` and `typespec-resource-manager`.

Reviewer: ```suggestion
> **NOTE:** Generating all languages from the same folder may be complicated due to the preview status of the emitters and TypeSpec libraries, such as `typespec-azure-core` and `typespec-azure-resource-manager`.
```

}
```

> To simplify the process of getting your code to emit without the need for multiple command line options, it's best to modify your `tspconfig.yaml` file.

Reviewer: ```suggestion
Always update your `tspconfig.yaml` file rather than passing options through the command line.
```

    package-name: azure-service-template
```

> Several language repositories utilize the `tsp-client` tool to simplify the process of generating client libraries. For more information on the tool, see [Getting started with `tsp-client`](<./../Generating with tsp-client/tsp_client.md>).

Reviewer: ```suggestion
Several language repositories utilize the `tsp-client` tool to simplify generating client libraries. For more information on the tool, see [Getting started with `tsp-client`](<./../Generating with tsp-client/tsp_client.md>).
```
